esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -830,12 +886,14 @@ def __init__(
             raise ValueError(msg.format(mdtol))
         self.mdtol = mdtol
         self.method = method
+        self.resolution = resolution
 
         partial_regrid_info = _regrid_rectilinear_to_unstructured__prepare(
             src_mesh_cube,
             target_grid_cube,
             method=self.method,
             precomputed_weights=precomputed_weights,
+            resolution=resolution,
```

Comment:
```suggestion
            resolution=self.resolution,
```

Less risk of future divergence.

Reply:
214dedaa8f3e48997731eca51182cb5d3080d9a3